### PR TITLE
[IMP] l10n_ve: Translation

### PR DESCRIPTION
**Pull request by malb-odoo**
All the localisation should be written in english and then translated to the native language of the country. In this case, the Venezuelian localisation was written is Spanish, this PR translate the module in english and add the corresponding PO file in spanish.

Task-id: 3126154

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ve/data/account.account.template.csv [hunk 1]
```diff
@@ -1,267 +1,267 @@
...
+"account_activa_account_1131003","MERCHANDISE INVENTORY VALUE UPDATE","1131003","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1131009","MERCHANDISE INVENTORY VALUE UPDATE","1131009","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1134001","GOODS IN TRANSIT","1134001","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1141001","ADVERTISING","1141001","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1141002","PREPAID INSURANCE","1141002","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1141003","H.C.M. INSURANCE","1141003","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1141004","RENTALS","1141004","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1151001","I.S.L.R. ESTIMATED STATEMENT","1151001","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_1151002","I.S.L.R.I. WITHHELD","1151002","asset_current","l10n_ve.ve_chart_template_amd","True"
```
Comment (vin-odoo) [inline addons/l10n_ve/data/account.account.template.csv:42]:
```suggestion
"account_activa_account_1151002","I.S.L.R. WITHHELD","1151002","asset_current","l10n_ve.ve_chart_template_amd","True"
```

File: addons/l10n_ve/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,1866 @@
...
+#: model:account.account.template,name:l10n_ve.account_activa_account_7151019
+msgid "DEDUCTIBLE TRAVEL EXPENSES"
+msgstr "VIATICOS DEDUCIBLES"
+
+#. module: l10n_ve
+#: model:account.account,name:l10n_ve.1_account_activa_account_1251001
+#: model:account.account,name:l10n_ve.2_account_activa_account_1251001
+#: model:account.account.template,name:l10n_ve.account_activa_account_1251001
+msgid "DEFERRED TAXES I.S.L.R."
+msgstr "IMPUESTOS DIFERIDOS I.S.L.R"
```
Comment (vin-odoo) [inline addons/l10n_ve/i18n/es.po:321]:
```suggestion
msgstr "IMPUESTOS DIFERIDOS I.S.L.R."
```

File: addons/l10n_ve/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,1866 @@
...
+#: model:account.tax.template,name:l10n_ve.tax0sale
+msgid "Exempt (sales)"
+msgstr "Exento (ventas)"
+
+#. module: l10n_ve
+#: model:account.account,name:l10n_ve.1_account_activa_account_2171002
+#: model:account.account,name:l10n_ve.2_account_activa_account_2171002
+#: model:account.account.template,name:l10n_ve.account_activa_account_2171002
+msgid "F.A.O.V. CONTRIBUTION COMPANY"
+msgstr "F.A.O.V APORTE EMPRESA"
```
Comment (vin-odoo) [inline addons/l10n_ve/i18n/es.po:481]:
```suggestion
msgstr "F.A.O.V. APORTE EMPRESA"
```

File: addons/l10n_ve/data/account.account.template.csv [hunk 4]
```diff
@@ -1,267 +1,267 @@
...
+"account_activa_account_7131013","BENEFITS SCHOLARSHIPS","7131013","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131016","SOCIAL BENEFITS","7131016","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131017","INTEREST ON SOCIAL BENEFITS","7131017","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131018","UTILITIES","7131018","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131019","HOLIDAYS","7131019","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131020","VACATION BONUS","7131020","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131021","CONTRIBUTIONS I.V.S.S.","7131021","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131022","F.A.O.V. CONTRIBUTIONS","7131022","asset_current","l10n_ve.ve_chart_template_amd","True"
+"account_activa_account_7131023","I.N.C.C.E.S. CONTRIBUTIONS","7131023","asset_current","l10n_ve.ve_chart_template_amd","True"
```
Comment (vin-odoo) [inline addons/l10n_ve/data/account.account.template.csv:186]:
```suggestion
"account_activa_account_7131023","I.N.C.E.S. CONTRIBUTIONS","7131023","asset_current","l10n_ve.ve_chart_template_amd","True"
```

**Review by vin-odoo — CHANGES_REQUESTED** [review]

**Comment by vin-odoo** [comment]
@robodoo r+

**Comment by malb-odoo** [comment]
@vin-odoo I forgot the spoken language in the chart template, can you R+ again ?

**Comment by vin-odoo** [comment]
@robodoo r+

**Pull request closed without merging**